src/server/index.js

```diff
@@ -39,12 +38,30 @@ import routes from './routes'
 
 app.use(routes)
 
-app.use(function *(next) {
-  this.body = 'Hello world'
-  yield next
-})
-
 const port = process.env.PORT || 3000
 app.listen(port, () => {
   debug(`Server running at http://localhost:${port}`, perf.stop(serverStart))
 })
+
+// Webpack Dev Server for hot reloading
+import webpack from 'webpack'
+import WebpackDevServer from 'webpack-dev-server'
+import config from '../../webpack.config.js'
+
+new WebpackDevServer(webpack(config), {
+  hot: true,
+  historyApiFallback: true,
+  proxy: {
+    '*': `http://localhost:${port}`
+  },
+  stats: {
+    colors: true
+  },
+  devtool: 'inline-source-map'
+}).listen(3001, 'localhost', (err, result) => {
+  if (err) {
+    debug(err)
+  }
+
+  debug(`${result}: Listening at http://localhost:3001`)
```

Comment:
Since I haven't really used this before, can you clue me in to what `result` is?